### ISSUE TITLE
NVDA not ignoring images in HTML emails with blank alt text

### ISSUE THREAD

**Issue by tombye** [alphagov/reported-bugs#46]
**Upstream bug:** https://github.com/nvaccess/nvda/issues/9874
**Date:** 2019-07-05
**Reported by:** Tom Byers
**Related to:** Not applicable

---

### Overview

Images with empty alt text in HTML emails should be treated like those in HTML web pages and ignored. When they appear in HTML emails, through the WIndows 10 Mail app, NVDA is not ignoring them and instead announces the URL of the image resource.

Steps to reproduce:
1. send a HTML email containing an image with an empty alt attribute
2. open and read the email in the Windows 10 Mail or Outlook 2016 using NVDA

### Expected result

The image should be ignored.

### Actual result

NVDA announces the `<img>` as 'graphic'.

**Comment by tombye** [comment]
I've updated the description of this to match the results from testing recently. More details in the upstream issue but TLDR: it doesn't announce the URL any more but still doesn't ignore it.

**Comment by tombye** [comment]
This issue has been closed by one of the team because they seem to believe the behaviour is expected:

https://github.com/nvaccess/nvda/issues/9874#issuecomment-1519071824

I've added a comment to check they understand my description of an image with `alt=""` rather than one **without** an alt attribute. Hopefully, this will clear up whether that is, in fact, what they intend to say.
